src/test/java/Homework21.java

```diff
@@ -0,0 +1,52 @@
+public class Homework21 extends LoginTests{
+
+
+    import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.TestNG;
+import org.testng.annotations.Test;
+
+    public class Homework20 extends LoginTests {
+
+    }
+//Prerequisite - at least one user created playlist
+
+        String newPlaylistName = "Sample Edited Playlist";
+
+        @Test
+        public void renamePlaylist() {
+
+            String updatedPlaylistMsg = "Updated playlist \"Sample Edited Playlist.\"";
+
+            provideEmail ("demo@class.com");
+            providePassword ("te$t$tudent");
+            clickSubmit ();
+            doubleClickPlaylist ();
+            enterNewPlaylistName ();
+            Assert.assertEquals (getRenamePlaylistSuccessMsg (), updatedPlaylistMsg);
```

Comment:
I don't see where you defile this getRenamePlaylistSuccessMsg function, please double check again